src/config.js

```diff
@@ -1,8 +1,20 @@
+const path = require("path");
+const os = require("os");
+
+const kopytoTempDir = '.kopytko_packager_temp';
+let tempDir;
+
+try {
+  tempDir = path.join(os.tmpdir(), kopytoTempDir);
```

Comment:
Could you add this info to the readme? We have there the `tempDir` definition in the [Configuration part](https://github.com/getndazn/kopytko-packager#configuration).